libraries/AP_Airspeed/AP_Airspeed.cpp

```diff
@@ -191,18 +192,33 @@ const AP_Param::GroupInfo AP_Airspeed::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("2_BUS",  20, AP_Airspeed, param[1].bus, 1),
     
+    // @Param: _BLEND
+    // @DisplayName: Blending of airspeed sensors
+    // @Description: Sets the configuration for enable the blending of multiple airspeed sensors
+    // @Values: 0:Blend disabled,1:Full blending
+    // @User: Advanced
+    AP_GROUPINFO("_BLEND",  21, AP_Airspeed, blend_conf, 0),
```

Comment:
I like that this is parameter controlled and disabled by default. This allows us to merge easier because it won't effect most people, only those who are poking around. Thank you!

Reply:
Exactly my point!